server/repository/src/db_diesel/clinician_row.rs

```diff
@@ -206,5 +198,15 @@ mod test {
 
         assert_eq!(repo.find_is_sync_update_by_id(&row.id), Ok(Some(false)));
         assert_eq!(repo.find_is_sync_update_by_id(&row2.id), Ok(Some(false)));
+
+        // Sneak in a test for insert_one_or_ignore. If another entry is inserted/upserted in
+        // insert_one_or_ignore() the clinician_id foreign key would be violated and the following
+        // would fail:
+        ClinicianLinkRowRepository::new(&connection)
```

Comment:
See code comment...

Reply:
Seems fine here to me, thanks for adding 🙏